app/controllers/homes_controller.rb

```diff
@@ -2,5 +2,6 @@ class HomesController < ApplicationController
   before_action :authenticate_user!
   
   def index
+    render :index
   end
 end
```

Comment:
this....does not make sense? remove `render :index` and this should work just fine